src/styles/videoosd.scss

```diff
@@ -174,6 +174,9 @@
 
 .osdMediaStatus {
     margin-left: auto;
+    display: flex;
+    align-items: center;
+    gap: 0.125rem;
```

Comment:
Gap isn't supported on some of the browser engines that we support. Please use equivalent margins instead.

Reply:
Thank you! I've made the necessary adjustment and replaced the gap.